src/autometrics/tracker/opentelemetry.py

```diff
@@ -184,3 +185,14 @@ def finish(
                     "module": module,
                 },
             )
+
+    def initialize_at_zero(
+        self,
+        function: str,
+        module: str,
+        objective: Optional[Objective] = None,
+    ):
+        """Initialize tracking metrics for a function call at zero."""
+        caller = ""
+        self.__count(function, module, caller, objective, None, Result.OK, inc_by=0)
```

Comment:
also nit, but i think since all other args are positional (vs keyword) we might want to go for consistency here

Reply:
fair enough! i think it's slightly less understandable at a glance without the name, but given the function name you can figure it out quickly enough